app/models/miq_widget.rb

```diff
@@ -571,6 +568,13 @@ def delete_legacy_contents_for_group(group)
     MiqWidgetContent.where(:miq_widget_id => id, :miq_group_id => group.id, :user_id => nil).destroy_all
   end
 
+  # default: timezone does matter
+  # options[:timezone_matters] == false will skip it
+  # TODO: detect date field in the report?
+  def timezone_matters?
+    options.try(:[], :timezone_matters) != false
```

Comment:
Wow, good point @kbrock.  Personally, I'd use a guard clause to make things very explicit:

Maybe something like this:
```ruby
def timezone_matters?
  return true unless options
  options.fetch(:timezone_matters, true)
end
```

If we ever make options default to an empty hash, we could easily remove that guard clause.